episodes/09-open-and-plot-vector-layers.Rmd

```diff
@@ -0,0 +1,188 @@
+---
+title: "Open and Plot Vector Layers"
+teaching: 25
+exercises: 5
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How can I read, examine and visualize point, line and polygon vector data in R?
```

Comment:
```suggestion
- How can you read, examine and visualize point, line and polygon vector data in R?
```

Reply:
It's better to frame it in a "you" perspective

Reply:
I think that 'I' works better in questions. It is the participant asking how he or she can do something.
In objectives, 'you' works well. Then it's the teacher explaining to the participant what he or she will learn, indicating that the lesson will provide answers to the participant's questions. 
(my 50 cent on this highly philosophical topic) ;)